go/stats/export.go

```diff
@@ -183,7 +183,9 @@ func publish(name string, v expvar.Var) {
 // to be pushed to it. It's used to support push-based metrics backends, as expvar
 // by default only supports pull-based ones.
 type PushBackend interface {
-	// PushAll pushes all stats from expvar to the backend
+	// Close shuts down the backend and flushes any pending data.
+	Close() error
```

Comment:
In order to reduce test flakiness, `--emit_stats` at a high frequency. In order to reduce data written to disk, use gzip compression, which requires calling `Close()` on the gzip writer to flush pending data. Integrate `PushBackend.Close` with `servenv.OnTerm`.